redisson/src/main/java/org/redisson/api/RAtomicDouble.java

```diff
@@ -106,6 +106,24 @@ public interface RAtomicDouble extends RExpirable, RAtomicDoubleAsync {
      * @param newValue the new value
      */
     void set(double newValue);
+    
+    /**
+     * Atomically sets the given value if current value is less than
+     * the special value
+     *
+     * @param less  compare value
+     * @param value newValue
+     */
+    void lessThanSet(double less, double value);
```

Comment:
I think it would be useful to return boolean for each method to indicate whether the value was updated.

Reply:
done